www/index.html

```diff
@@ -95,72 +95,219 @@
+                                <code>index.html</code> from the file system).
+                            </li>
+                        </ul> 
+                    </p>
+                    <p style="text-align: right"><a href="#contents">↑ Back to Contents</a></p>
+                    
+                    <h3 id="feedback">Feedback / helping / contributing</h3>
+                    <p>This application is still work in progress. There may be bugs and issues, and new features are planned.
+                        Suggestions and patches/pull requests are welcome : the source code is on <a href="https://github.com/kiwix/kiwix-js" target="_blank">github</a>.
+                    </p>
+                    <p>
+                        The <a href="https://github.com/kiwix/kiwix-js/issues" target="_blank">bugtracker</a> is on github too. We use it as our roadmap.
+                        <br />Alternatively, you can send your feedback <a href="mailto:mossroy@mossroy.fr?subject=%5BKiwix%5D%20Feedback%20about%20Kiwix%20JS">by email</a>.
+                        <br />In all cases, please give as much detail as you can : which device you are using, which version of Kiwix, which archive, which articles etc.
+                        <br />
+                    </p>
+                    <p>
+                        If you like the Kiwix project and use it, please consider giving donations :
+                        <ul>
+                            <li>to the Wikimedia foundation, to help maintaining their servers and article contents. See the <a href="https://donate.wikimedia.org/" target="_blank">donation page</a></li>
+                            <li>to Kiwix, to help maintaining the ZIM files and applications. There is a donation button in  the <a href="https://www.kiwix.org">home page of kiwix.org</a></li>
+                        </ul>
+                        You can also help by :
+                        <ul>
+                            <li>Adding articles or improving the existing ones, by <a href="https://en.wikipedia.org/wiki/Wikipedia:Introduction" target="_blank">editing online wikipedia articles</a></li>
+                            <li><a href="https://wiki.kiwix.org/wiki/Special:MyLanguage/Participate" target="_blank">Participating</a> in the Kiwix project</li>
+                        </ul>
+                    </p>
+                    <p style="text-align: right"><a href="#contents">↑ Back to Contents</a></p>
+                    
+                    <h3 id="credits">Credits</h3>
+                    <p>We owe a big debt of gratitude to:
+                        <ul>
+                            <li>Kelson and all the Kiwix community for welcoming us in this project</li>
+                            <li>Wikipedia teams and contributors to help spreading knowledge to everybody</li>
+                            <li>All the developers of libraries and tools we used for releasing them as Free software (see Licence paragraph)</li>
```

Comment:
We might add a coma, like :
```suggestion
                            <li>All the developers of libraries and tools we used, for releasing them as Free software (see Licence paragraph)</li>
```